kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/resources/form-templates-providers/patternfly/input-form-group-template.html

```diff
@@ -62,6 +61,81 @@
+    </div>
+</div></div></div>
+    <div data-field="helpBlock" class="help-block"></div>
+</div>
+<#elseif item.type == "documentCollection">
+<div>
+	<label for="${item.id}">${item.label}</label> 
+    <div id="${item.id}" data-field="documentCollection">
+    	<#if item.value?size gt 0>
+    	    <#assign documents=item.value>
+			<#list documents as file>
+			<div class="document"> 
+				<span data-field="state"></span>
+				<span class="kie-wb-common-forms-docs-upload-content vertical-">
+				        <a data-field="document" target="_blank" data-name="${file.name}" data-bytes="${file.content}">${file.name}</a>
+				</span>
+				<span data-field="actions" class="kie-wb-common-forms-docs-upload-content"><span aria-hidden="true" class="close">×</span></span>
+			</div>
+			</#list>
+		    <div class="kie-wb-common-forms-docs-upload" data-field="dropRegion" >
+			<div class="kie-wb-common-forms-docs-upload-dropHere"  data-field="dropHere">
+	            <div class="kie-wb-common-forms-docs-upload-dropHere-content">
+	                <h3 data-i18n-key="dropFilesHere">Drop files here</h3>
+	            </div>
+	        </div>
+	        </div>
+		<#else>
+		    <div class="kie-wb-common-forms-docs-upload" data-field="dropRegion" >
+	        	<div class="kie-wb-common-forms-docs-upload-empty" data-field="emptyState">
+	            	<div class="kie-wb-common-forms-docs-upload-empty-content"  data-field="dragHere" >
+		                <h3>
+		                    <span data-i18n-key="dragFilesText">Drag files here to upload</span>
```

Comment:
It should match with what it is designed in BC - I mean, form is allowing to upload files not only by drag&drop but also using a file browse through an upload button.

Reply:
not really.